R/repr_packageIQR.r

```diff
@@ -0,0 +1,68 @@
+#' packageIQR representations
+#'
+#' plain text representations of packageIQR objects like the list of vailable example
+#' data or vignettes
+#'
+#' @param obj  The packageIQR obj to create a representation for
+#' @param ...  ignored
+#'
+#' @name repr_*.packageIQR
+#' @export
+repr_text.packageIQR <- function(obj, ...)
+{
+  # this is mostly copied from utils:::print.packageIQR
+  db <- obj$results
+  out <- if (nrow(db) > 0L) {
+	lapply(split(seq_len(nrow(db)), db[, 'Package']),
+		   function(ind) {
```

Comment:
indentation is super weird now. can you ensure that regex-searching `^\t*[ ]` has no matches?